cli/ops.rs

```diff
@@ -69,9 +69,9 @@ fn empty_buf() -> Buf {
 }
 
 /// Processes raw messages from JavaScript.
-/// This functions invoked every time libdeno.send() is called.
-/// control corresponds to the first argument of libdeno.send().
-/// data corresponds to the second argument of libdeno.send().
+/// This functions invoked every time Deno.core.dispatch() is called.
```

Comment:
@ry Shouldn't this be `Deno.core.send`?

Reply:
I need to add some docs, but Deno.core.dispatch is the higher-level version of Deno.core.send. So while it's true this will be invoked for every Deno.core.send, users will only be calling Deno.core.dispatch...

Reply:
I think we're still iterating on how exactly this works, so I think it's okay to be a bit vague at the moment. But we need to nail this down soon.

Reply:
All clear, thanks